Cargo.toml

```diff
@@ -20,3 +20,6 @@ serde_json = "^1.0.107"
 strum = "0.25.0"
 strum_macros = "^0.25.3"
 typetag = "0.2.13"
+
+[dev-dependencies]
+rstest = "0.18"
```

Comment:
Fixtures, learnt something new

Reply:
Yeah, although I feel like all rust test libs leave much to be desired..

`rstest` works well for parametrisation, though. After I gave Claude a crash course on converting things to `rstest` macros, it converted all this seamlessly 😄 
https://github.com/lballabio/QuantLib/blob/master/test-suite/daycounters.cpp#L849

Reply:
yh Claude is a stand up guy